di_website/templates/events/event_page.html

```diff
@@ -0,0 +1,71 @@
+{% extends "layouts/default.html" %}
```

Comment:
Like I said, check the patterns library template to make sure you're extending the correct layout.

Reply:
Template now updated to sidebar.html